src/components/AdvancedSearchForm/AdvSearchDropDownItem.vue

```diff
@@ -1,9 +1,16 @@
 <template>
-  <DropDownItem class="!whitespace-nowrap overflow-ellipsis overflow-x-hidden">
+  <DropDownItem
```

Comment:
There's probably a couple things we want to do here. I think there's a bug in the old UI where collections not marked as browsable still show in the advanced search list. 

We also never want to reveal the existence of a collection that a user doesn't have permission to view. So we don't want to disable those, they just shouldn't exist as far as that user is concerned.

So this list should basically be a mirror the "list collections" page - only collections which are "browsable" and which the user has view permissions to, I think. So that'd basically mirror what we show on the list collections page?